### ISSUE TITLE
[BUG] - LANDING GEAR BOGEY TILT 

### ISSUE THREAD

**Issue by PANAM741** [invernyx/md11-bugs#1427]
### Describe the bug

I noticed the landing gear angles are fixed. It must be able tilt. All aircraft types.

### Expected behaviour

1. Gear level with aircraft when airborne.
2. Gear level with the runway when in contact with the runway.
3. Transitions between the two mentioned above.

The gear will be able to tilt so its aligned with the ground

### Evidence

![Screenshot_433](https://github.com/invernyx/md11-bugs/assets/71863590/28a15aec-b70b-4a0f-9145-4b8c9bfa353a)
![IMG_5051](https://github.com/invernyx/md11-bugs/assets/71863590/7b2abe2e-d599-41fd-bd84-9d614c99de1d)
![IMG_5028](https://github.com/invernyx/md11-bugs/assets/71863590/2477579a-ba22-46f2-9a89-5d48a6fde306)

### Simulator

Microsoft Flight Simulator 2020 (MSFS)

### Aircraft Type

Passenger GE

### Agreement

- [X] Yes

**Comment by Mets747** [comment]
+1

**Comment by arwasairl** [comment]
@jhattermann Can you look into this; I'm not super familiar with the gear rig as you did it originally.

**Comment by jhattermann** [comment]
Wonder if we can "steal" some behavior XML from the default 748

**Comment by edueocara27** [comment]
I was about to report this. The funny thing is that the second image that shows the correct tilt of the MD-11 main landing gear is from Rotate MD-11 in X-Plane. When it was released, it had exactly the same bug, which was later fixed, as you can see in the screenshot.

I believe this is a relatively simple bug to fix.

![Microsoft Flight Simulator Screenshot 2024 07 04 - 20 43 09 36](https://github.com/invernyx/md11-bugs/assets/79176307/be4beab2-04e7-4641-be06-ca3f950db5f0)

**Comment by Mets747** [comment]
Yeah the gear animation is totally wrong, the image above all wheels should be making contact with the ground. as soon as rotation begins is when they should have the boogey tilt. PMDG nailed it with their 777....

**Comment by GPUthesteve** [comment]
> Wonder if we can "steal" some behavior XML from the default 748

no need, there's one from fsx era, might be useful (animated from frame 0-100, 50 is "on ground, no tilt" afaik)
<AnimLength>100</AnimLength>
    <Animation>
      <Parameter>
        <Code>(A:SIM on GROUND, bool) 1 == if{ 50 } els{ (A:PLANE PITCH DEGREES, degree) /-/ 50 + } </Code>
      </Parameter>
    </Animation>

**Comment by Mets747** [comment]
Has this been looked at? This is a hugeeee immersion killer for myself and many others.
